jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainer.java

```diff
@@ -59,7 +59,10 @@ public boolean equals(Object other) {
     if (this == other) {
       return true;
     }
-    if (!(other instanceof JibContainer)) {
+    if (other == null) {
```

Comment:
I think we can combine these two into `if (other == null || getClass() != other.getClass())`.